Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py

```diff
@@ -305,27 +409,92 @@ def list_findings(client: boto3.client, args: dict):
                           outputs_key_field='')
 
 
+def get_pagination_args(args: dict):
+    """
+    Gets and validates pagination arguments.
+    :param args: The command arguments (page, page_size or limit)
+    :return: limit, page_size, page after validation and convert
+    """
+
+    # Automatic Pagination
+    limit = arg_to_number(args.get('limit', MAX_RESULTS_RESPONSE))
+
+    # Manual Pagination
+    page = arg_to_number(args['page']) if args.get('page') else None
+    if page is not None and page <= 0:
+        raise Exception('page argument must be greater than 0')
+
+    page_size = arg_to_number(args['page_size']) if args.get('page_size') else MAX_RESULTS_RESPONSE
+    if not 0 < page_size <= MAX_RESULTS_RESPONSE:  # type: ignore
+        raise Exception(f'page_size argument must be between 1 to {MAX_RESULTS_RESPONSE}')
+
+    if page:
+        limit = page * page_size  # type: ignore
+
+    return limit, page_size, page
+
+
+def parse_finding(finding: dict):
```

Comment:
Please add return type and a short docstring about what the function does as you did in `get_pagination_args`